txpipe/rail_pz.py

```diff
@@ -197,3 +270,74 @@ def write_output_chunk(self, output_file, start, end, z, pz_data):
         output_file["pdf/pdf"][start:end] = p
         output_file["point_estimates/z_mode"][start:end] = pz_data["zmode"]
         output_file["point_estimates/z_mean"][start:end] = mu
+
+
+
+class PZRailEstimateSource(PZRailEstimateLens):
```

Comment:
is there a logic whether the source or the lens is the parent class in estimate vs. train?

Reply:
There wasn't, so I've harmonized this a bit.